src/utilities/find_ML_normfactors3D.cxx

```diff
@@ -114,7 +114,8 @@ main(int argc, char** argv)
                                             do_block,
                                             do_symmetry_per_block,
                                             do_KL,
-                                            do_display);
+                                            do_display,
```

Comment:
This call needs to be rewritten in terms of the class, such that we don't use the deprecated function anymore.